checks/handlerCheck.go

```diff
@@ -2,93 +2,49 @@ package checks
 
 import (
 	"fmt"
-	"os"
 	"strings"
 
 	"github.com/newrelic/newrelic-lambda-extension/config"
 	"github.com/newrelic/newrelic-lambda-extension/lambda/extension/api"
 	"github.com/newrelic/newrelic-lambda-extension/util"
 )
 
-type RuntimeHandlerCheck interface {
-	check(handlerConfigs) error
-}
-
-type wrapperCheck struct {
-	wrapperName string
-	fileType    string
-}
-
 type handlerConfigs struct {
 	handlerName string
 	conf        *config.Configuration
 }
 
-var handlerCheck = map[string]RuntimeHandlerCheck{
-	"node": wrapperCheck{
-		wrapperName: "newrelic-lambda-wrapper.handler",
-		fileType:    "js",
-	},
-	"python": wrapperCheck{
-		wrapperName: "newrelic_lambda_wrapper.handler",
-		fileType:    "py",
-	},
-}
-
-var pathExists = func(p string) error {
-	if _, err := os.Stat(p); os.IsNotExist(err) {
-		return err
-	}
-	return nil
-}
-
-var checkAndReturnRuntime = func() RuntimeHandlerCheck {
-	for k, v := range handlerCheck {
-		p := fmt.Sprintf("/var/lang/bin/%s", k)
-		err := pathExists(p)
-
-		if err == nil {
-			return v
-		}
-	}
-	// If we make it here that means the runtime is not one we
-	// currently validate so we don't want to warn against anything
-	return nil
-}
-
-const handlerPath = "/var/task"
+var handlerPath = "/var/task"
```

Comment:
Should the logic surrounding this var and the path construction be rolled into the runtime config?

Reply:
I considered the same thing but ended up leaving it here because it is handler related and not runtime related. But if you feel strongly about it I'm happy to move it.

Reply:
Don't have a strong opinion here.